File: 001-sum-of-array-elements.js
function findSum(arr) {
    const n = arr.length;

    let sum = 0;
    for(let element of arr) {
        sum += element;
    }

    return sum;
}

function main() {
    const arr = [10, 20, 30, 40, 50];
    const sum = findSum(arr);
    console.log(`Sum = ${sum}`);
}

main();

File: 001_SumOfArrayElements.java
package arrays;

class SumOfArrayElements {

    public static int findSum(int[] arr) {
        int n = arr.length;

        int sum = 0;
        // for(int i=0; i<n; i++) {
        //     sum += arr[i];
        // }

        for(int element: arr) {
            sum += element;
        }

        return sum;
    }
    public static void main(String[] args) {
        int[] arr = {10, 20, 30, 40, 50};

        int sum = findSum(arr);
        System.out.printf("Sum = %d%n", sum);
    } 
}
